generators/client/templates/react/webpack/webpack.common.js.ejs

```diff
@@ -75,7 +75,7 @@ module.exports = options => ({
         test: /\.tsx?$/,
         use: getTsLoaderRule(options.env),
         include: [utils.root('./src/main/webapp/app')],
-        exclude: /'node_modules'/
+        exclude: [path.resolve(__dirname, "node_modules")]
```

Comment:
I think utils.root does the same, can you check and use that method

Reply:
ok! let me check.